executor/batch_point_get.go

```diff
@@ -331,6 +332,7 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 				tID = getPhysID(e.tblInfo, d.GetInt64())
 			}
 		}
+		e.memTracker.Consume(int64(cap(handle.Encoded())))
```

Comment:
`IntHandle.Encoded()` will cause more overhead.
I think maybe we need a `kv.Handle.MemoryUsage()` to deal with the different kinds of handles.

Besides, `e.handles` will change in following codes (see line no. 374 & 392). So it would be better to put this codes after line no. 374 `handles = append(handles, e.handles[i])`.

Reply:
There is no kv.Handle.MemoryUsage()...

Reply:
I meant maybe we can add such a method.

Reply:
done